music_assistant/server/providers/fully_kiosk/__init__.py

```diff
@@ -146,22 +146,22 @@ async def loaded_in_mass(self) -> None:
     def _handle_player_update(self) -> None:
         """Update FullyKiosk player attributes."""
         player_id = self._fully.deviceInfo["deviceID"]
-        player = self.mass.players.get(player_id)
-        player.name = self._fully.deviceInfo["deviceName"]
-        # player.volume_level = snap_client.volume
-        for volume_dict in self._fully.deviceInfo.get("audioVolumes", []):
-            if str(AUDIOMANAGER_STREAM_MUSIC) in volume_dict:
-                volume = volume_dict[str(AUDIOMANAGER_STREAM_MUSIC)]
-                player.volume_level = volume
-                break
-        current_url = self._fully.deviceInfo.get("soundUrlPlaying")
-        player.current_item_id = current_url
-        if not current_url:
-            player.state = PlayerState.IDLE
-        player.available = True
-        self.mass.players.update(player_id)
-
-    async def get_player_config_entries(self, player_id: str) -> tuple[ConfigEntry]:
+        if player := self.mass.players.get(player_id):
```

Comment:
I've made a bunch of those fail silently on the assumption that such the player being missing would happen when we were unloading a player. Maybe raising `PlayerUnavailableError` would be better in some cases? I'm not sure.

Reply:
mass.players.get has an optional argument to raise on unavailable but ofc mypy has no knowledge of that

I think just doing this None check is fine - probbaly really some edgecases we're talking about